content/docs/tasks/telemetry/distributed-tracing/zipkin/index.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
@wu-sheng it is actually this page which is needing translation. you can see below a translation for the jaeger variant was moved.

optional is screen shot with chinese text in zipkin (right now images show English but if in china zipkin would show chinese labels)